docs/book/src/topics/api-server-endpoint.md

```diff
@@ -4,20 +4,21 @@ This document describes how to configure your clusters' api server load balancer
 
 ### Load Balancer Type
 
-CAPZ supports two load balancer types, `Public` and `Internal`. `Public`, which is also the default, means that your API Server Load Balancer will have a publicly accessible IP address.
-`Internal`, also known as a "private cluster", means that the API Server endpoint will only be accessible from within the cluster's virtual network (or peered VNets).
+CAPZ supports two load balancer types, `Public` and `Internal`.
 
-A `Public` cluster will have an Azure public load balancer load balancing internet traffic to the control plane nodes.
+`Public`, which is also the default, means that your API Server Load Balancer will have a publicly accessible IP address. This Load Balancer type supports a "public cluster" configuration, which load balances internet source traffic to the apiserver across the cluster's control plane nodes.
 
-A `Private` cluster will have an Azure internal load balancer load balancing traffic inside the VNet to the control plane nodes.
+`Internal` means that the API Server endpoint will only be accessible from within the cluster's virtual network (or peered VNets). This configuration supports a "private cluster" configuration, which load balances internal VNET source traffic to the apiserver across the cluster's control plane nodes.
+
+For a more complete "private cluster" template example, you may refer to [this reference template](../../../../templates/cluster-template-private.yaml) that the capz project maintains.
```

Comment:
this url won't be accessible from the book, it can only reference fully resolved URLs

```suggestion
For a more complete "private cluster" template example, you may refer to [this reference template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/cluster-template-private.yaml) that the capz project maintains.
```